src/tools/spectest-interp.cc

```diff
@@ -1807,6 +1833,18 @@ wabt::Result CommandRunner::OnAssertExhaustionCommand(
   return wabt::Result::Ok;
 }
 
+wabt::Result CommandRunner::OnAssertExceptionCommand(
+    const AssertExceptionCommand* command) {
+  ActionResult result =
+      RunAction(command->line, &command->action, RunVerbosity::Quiet);
+  if (!result.trap || result.trap->message() != "uncaught exception") {
+    PrintError(command->line, "expected an exception to be thrown");
+    return wabt::Result::Error;
+  }
+
```

Comment:
```suggestion
PrintError(command->line, "assert_exception passed");
```
Would it be good to print this as in `assert_trap`? https://github.com/WebAssembly/wabt/blob/0dfa83c1bc645eaa65d8e6ee51f42f8578931ecc/src/tools/spectest-interp.cc#L1788-L1789

Reply:
Good idea, thanks. I updated the tests to match this printing.